? 0001-Token-and-CCK-conditions-for-additional-descriptions.patch
? 1025716-broke-empty-fields.patch
? 806874-render-first.patch
? 806874-render-first_0.patch
? 827628-content-with-same-title-disappearing.patch
? 869766-all-left-fixed-flexible-broken.patch
? 877810-add-flexible.patch
? 903216-legacy-needs-add_css.patch
? 906520-ie7-problems-with-rounded-shadows.patch
? 920266-panels-simple_cache-t.patch
? 9439310-render_pane-inconsistent.patch
? 945218-node-body-not-required-with-no-word-count.patch
? 955102-notice-in-panel-nodes.patch
? 958088-legacy-not-adding-css.patch
? 964334-breaks-views-row-styles.patch
? admin-links-2.patch
? body_field3.patch
? button-position.patch
? cache_languge.patch
? destroy-sort.patch
? flexible_css_1.patch
? panels-917614.patch
? panels-932632.patch
? panels-add-value-to-requirements-ok.patch
? panels-help-layout.patch
? panels-simple-cache-pager.patch
? panels.panels_update_6310.patch
? panels_fix_css_cleanstring.patch
? panels_ipe-887444-5.patch
? panels_ipe-888410.patch
? panels_ipe_css1.patch
? panels_ipe_i18n.patch
? sa-2754-panels-css-ids.patch
? sa-unscrubbed-class-and-id.patch
? undo.patch
? panels_ipe/TODO.txt
? plugins/layouts/flexible960
Index: plugins/views/panels.views.inc
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/panels/plugins/views/panels.views.inc,v
retrieving revision 1.1.2.3
diff -u -p -r1.1.2.3 panels.views.inc
--- plugins/views/panels.views.inc	16 Dec 2010 06:27:52 -0000	1.1.2.3
+++ plugins/views/panels.views.inc	15 Jan 2011 00:54:51 -0000
@@ -11,7 +11,8 @@ function panels_views_plugins() {
         'help' => t('Displays the fields in a panel rather than using a template.'),
         'handler' => 'panels_views_plugin_row_fields',
         'path' => drupal_get_path('module', 'panels') . '/plugins/views',
-        'theme' => NULL,
+        'theme' => 'views_view_fields',
+        'theme path' => drupal_get_path('module', 'views') . '/theme',
         'uses fields' => TRUE,
         'uses options' => TRUE,
         'type' => 'normal',
